resolwe_bio/processes/reads_processing/xengsort.py

```diff
@@ -156,7 +147,7 @@ class Advanced:
     class Output:
         """Output fields to process XengsortIndex."""
 
-        index = FileField(label="Xengsort index")
+        index = DirField(label="Xengsort index")
```

Comment:
I explored that initially, but decided to go with the DirField so that we can use a single input reference (pointing to the dir + common index file prefix). By having the two files stored in different output fields, we would need to use symlink in the classify process, which is not so nice.